docs/release/trg-1/trg-1-08.md

```diff
@@ -0,0 +1,26 @@
+---
+title: TRG 1.08 - Interface documentation (APIs)
+sidebar_position: 1
+---
+
+| Status | Created     | Post-History                                          |
+|--------|-------------|-------------------------------------------------------|
+| Active | 22-Apr-2024 |                                                       |
+| Draft  | 21-Mar-2024 | Transferred from Release Acceptance Criteria into TRG |
+
+## Why
+
+Interface documentation is crucial to interact with components. It fundamentally enables others to consume your APIs.
+
+## Description
+
+- The interface documentation **must** be stored in the folder `/docs/api` as `openAPI.yaml`
+  - It **must** be in the OpenAPI standard
```

Comment:
which version?

Reply:
I clarified it. It as to be at least 2. I assume this is reasonable enough as 2 is default for a while